docs/source/additional-resources/index.rst

```diff
@@ -3,13 +3,14 @@ Additional Resources
 
 .. _dbnotebooks:
 
-Running Databricks notebooks
-----------------------------
+Databricks notebooks
+--------------------
 
-Most of the code in the Databricks notebooks can be run on Spark and Glow alone, but some functions are only available
-on Databricks. `Try Databricks for free <https://databricks.com/try-databricks>`_ and import the Databricks notebooks
-following the
-`Databricks documentation <https://docs.databricks.com/notebooks/notebooks-manage.html#import-a-notebook>`_.
+Most of the code in the Databricks notebooks can be run on Spark and Glow alone, but some functions are only available on Databricks.
+
+.. note::
+
```

Comment:
I would create another heading here instead of a note.. something like: "New to Databricks? Try Glow on Databricks for Free!" and the have a line in that section like "Databricks Community Edition is free. Follow <link to instructions> to try Glow and all notebooks in this documentation on it.